### ISSUE TITLE
number of reports not calculated for parse cmd

### ISSUE THREAD

**Issue by gyorb** [Ericsson/codechecker#2539]
running `CodeChecker parse -e html reports -o html_reports` will convert the reports to html
but the `Total number of reports: 0` even if there were reports.
```
----==== Summary ====----
----=================----
Total number of reports: 0
----=================----

----======== Statistics ========----
-------------------------------------
Number of processed plist files |  90
Number of analyzer reports      | 246
-------------------------------------

----==== Checker Statistics ====----
----------------------------------------------------------------
Checker name                                 | Number of reports
----------------------------------------------------------------
bugprone-misplaced-widening-cast             |                 2
bugprone-sizeof-expression                   |                58
cert-err09-cpp                               |                37
clang-diagnostic-implicitly-unsigned-literal |                 7
```
